.ipynb_checkpoints/wdfReader-checkpoint.py

```diff
@@ -0,0 +1,264 @@
+from __future__ import print_function
```

Comment:
is this file needed in the version control?

Reply:
same as above